src/EditorFeatures/VisualBasicTest/Diagnostics/UseAutoProperty/UseAutoPropertyTests.vb

```diff
@@ -553,5 +553,33 @@ end class")
     Public Property [|P|] As Integer
 End Class")
         End Function
+
+        <WorkItem(23735, "https://github.com/dotnet/roslyn/issues/23735")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)>
+        Public Async Function ExplicitInterfaceImplementation() As Task
+            Await TestMissingInRegularAndScriptAsync("
```

Comment:
Why are we disabling things for VB?  This is only a problem for C# right?

Reply:
You are right. I changed the PR to suppress the fixer for C# only.